test/fixtures/incorrect.js

```diff
@@ -88,6 +88,11 @@ class NoDupeClassMembers {
 
 noop(NoDupeClassMembers);
 
+// `no-dupe-keys`.
+const noDupeKeys = { foo: 'bar', foo: 'bar' };
```

Comment:
```suggestion
const noDupeKeys = { foo: 'bar', foo: 'rab' };
```

Optional, perhaps a tiny suggestion.. to draw attention to the fact that it's the key duplication that's invalid, rather than having the same value twice in different keys

Reply:
✅